### DISCUSSION TITLE
Default Custom Fields

### DISCUSSION THREAD

**Discussion by moritztim** [dani-garcia/vaultwarden#3812] [Ideas]
If it's possible, it would be great to create custom fields that show up by default when the user creates a new entry.

**Comment by BlackDex** [comment]
Not possible from the server-side. And since Vaultwarden is actually only the server side, it's not possible. Also, if possible, then we would only we able to do this on the web-vault and not other clients.

**Reply by moritztim** [reply]
That's true, I also checked bitwarden's issue tracker and part of the idea has already been suggested and added to the roadmap there.
